src/com/ableton/VirtualEnv.groovy

```diff
@@ -37,24 +37,29 @@ class VirtualEnv implements Serializable {
    * @return New instance of VirtualEnv object.
    */
   static VirtualEnv create(Object script, String python, String pyenvRoot) {
-    if (!script.isUnix()) {
-      script.error 'This method is not supported on Windows'
-    }
-
     VirtualEnv venv = new VirtualEnv(script)
     assert pyenvRoot
 
     if (!script.fileExists(pyenvRoot)) {
       script.error "pyenv root path '${pyenvRoot}' does not exist"
     }
 
-    List activateCommands = []
+    // Cache for later usage
+    boolean isUnix = script.isUnix()
+    String safePyenvRoot = isUnix ? pyenvRoot : convertWindowsPath(pyenvRoot)
+    List activateCommands = ["export PATH=${safePyenvRoot}/bin:\$PATH"]
```

Comment:
I'm not sure I see it. Maybe I'm missing something obvious? Either way, would it be ok to squash so that there aren't so many fixups cluttering the review?

Reply:
Ok, now that I've squashed and looked at the commits in the branch, I still don't understand this comment. Can you re-review and clarify please?

Reply:
In 31ed1db0d80e8133d34fa78d5a06cca5cffa475b, the old version had `export PATH=...`, the new version doesn't,

Reply:
Ah, ok, now I see it. Please see 7812313.